<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CurveSettings</class>
 <widget class="QWizardPage" name="CurveSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>283</width>
    <height>162</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WizardPage</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>GroupBox</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::ExpandingFieldsGrow</enum>
      </property>
      <item row="1" column="0">
       <widget class="QLabel" name="maximumNoiseLabel">
        <property name="text">
         <string>Maximum noise</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="maximumNoiseDoubleSpinBox">
        <property name="suffix">
         <string> a.u.</string>
        </property>
        <property name="decimals">
         <number>5</number>
        </property>
        <property name="singleStep">
         <double>0.001000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="averageNoiseLabel">
        <property name="text">
         <string>Average noise</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="averageNoiseDoubleSpinBox">
        <property name="suffix">
         <string> a.u.</string>
        </property>
        <property name="decimals">
         <number>5</number>
        </property>
        <property name="singleStep">
         <double>0.001000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="deltaEpsilonLabel">
        <property name="text">
         <string>Delta epsilon</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="deltaEpsilonDoubleSpinBox">
        <property name="suffix">
         <string> a.u.</string>
        </property>
        <property name="decimals">
         <number>5</number>
        </property>
        <property name="singleStep">
         <double>0.001000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
